### PR TITLE
add captcha template

### PR DESCRIPTION
**Pull request by ann26**
fix https://github.com/kartoza/LEDET_BIMS/issues/10

### CODE REVIEW

**Comment by timlinux** [comment]
Thanks @ann26  - can you share a GIF of it working?

File: bims/templates/includes/captcha.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<div id="captcha-info"><span id="warning-captcha">Please verify that you are not a bot first.</span><br/></div>
+<div class="g-recaptcha" data-callback="enableBtn"></div>
+
+{% if site_key != '' %}
```
Comment (dimasciput) [inline bims/templates/includes/captcha.html]: Where is this come from?

File: bims/templates/includes/captcha.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+<script type="text/javascript">
+    var CaptchaCallback = function(){
+      $('.g-recaptcha').each(function(){
+        grecaptcha.render(this,{'sitekey' : '{{ site_key }}' } );
+      })
+    };
+
+    function enableBtn(){
+        $('.btn-login').show();
```
Comment (dimasciput) [inline bims/templates/includes/captcha.html]: This means that this template can't be reused for other form, maybe create a function here to search a button that has specific class name e.g `btn-need-recaptcha`.

File: bims/templates/includes/captcha.html (hunk 1 above)
Comment (ann26) [reply bims/templates/includes/captcha.html]: I fixed it, using context processor

File: bims/templates/includes/captcha.html (hunk 2 above)
Comment (ann26) [reply bims/templates/includes/captcha.html]: done

**Comment by ann26** [comment]
@dimasciput here is captcha template test
peek 2018-06-06 14-40

**Comment by dimasciput** [comment]
LGTM

**Pull request merged**